common/src/main/resources/assets/homecamp/lang/pl_pl.json

```diff
@@ -0,0 +1,14 @@
+{
+    "modmenu.descriptionTranslation.homecamp": "${mod_description}",
```

Comment:
English `${mod_description}` for this one is "Make a Campfire your respawn point."